pkg/virt-operator/resource/apply/delete.go

```diff
@@ -500,13 +501,52 @@ func DeleteAll(kv *v1.KubeVirt,
 		}
 	}
 
+	if err = deleteKubeVirtLabelsFromNodes(clientset); err != nil {
+		return err
+	}
+
 	err = deleteDummyWebhookValidators(kv, clientset, stores, expectations)
 	if err != nil {
 		return err
 	}
 	return nil
 }
 
+func deleteKubeVirtLabelsFromNodes(clientset kubecli.KubevirtClient) error {
+	nodeList, err := clientset.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{LabelSelector: "kubevirt.io/schedulable"})
```

Comment:
Can you please use the constant [NodeSchedulable](https://github.com/kubevirt/kubevirt/blob/main/staging/src/kubevirt.io/api/core/v1/types.go#L893) instead of the hardcoded string?

Reply:
@alicefr done